bucket/windowsdesktop-runtime.json

```diff
@@ -1,39 +1,54 @@
 {
-    "version": "3.1.10",
-    "description": "Microsoft .NET Core Desktop Runtime",
-    "homepage": "https://dotnet.microsoft.com/download/dotnet-core",
+    "version": "5.0.4",
+    "description": "Microsoft .NET 5.0 Desktop Runtime",
+    "homepage": "https://dotnet.microsoft.com/download/dotnet",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx"
     },
     "notes": "You can now remove this installer with 'scoop uninstall windowsdesktop-runtime'",
     "architecture": {
         "64bit": {
-            "url": "https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.10/windowsdesktop-runtime-3.1.10-win-x64.exe",
-            "hash": "sha512:14a7d97fb258bee024cff3585492ed43c3ec6ac823b50980ddde8241a8bca3e578c38ca28e461630d38c180bd72323e1fcb0ee2e6e65ef9bfc8481df7beef142"
+            "url": "https://dotnetcli.blob.core.windows.net/dotnet/WindowsDesktop/5.0.4/windowsdesktop-runtime-5.0.4-win-x64.exe",
+            "hash": "sha512:163c6b9810e8aa66dca7caf20e4052df41d8ca844932ae1fcdcee3207f242f9fa455a976720c7e1e8473a19cf413d593ae46fe46fddee5ed458d2c97695dbb9d"
         },
         "32bit": {
-            "url": "https://dotnetcli.azureedge.net/dotnet/Runtime/3.1.10/windowsdesktop-runtime-3.1.10-win-x86.exe",
-            "hash": "sha512:1697e69644a1df46922e440c1ea16e583660f79f3b9b109b493a24835ae543ea6180c5167d6dbe89d8dd8991f865dba717525e333e4a2f32319733afa6a320b5"
+            "url": "https://dotnetcli.blob.core.windows.net/dotnet/WindowsDesktop/5.0.4/windowsdesktop-runtime-5.0.4-win-x86.exe",
+            "hash": "sha512:b0bd45e1d234ad74e770a7e1acea527f9e84af39da2cdecc6b4ab5c3f855b7f90d5ce7114f1500a9f9163de125cb87a6e4eb99a35d03ba5369db163cce8e3a87"
+        },
+        "arm64": {
```

Comment:
????

Reply:
?

Reply:
So the checking script won't recognise arm64? Fine...